rclpy/rclpy/impl/logging_severity.py

```diff
@@ -31,3 +31,15 @@ class LoggingSeverity(IntEnum):
     WARN = _rclpy.RCUTILS_LOG_SEVERITY.RCUTILS_LOG_SEVERITY_WARN
     ERROR = _rclpy.RCUTILS_LOG_SEVERITY.RCUTILS_LOG_SEVERITY_ERROR
     FATAL = _rclpy.RCUTILS_LOG_SEVERITY.RCUTILS_LOG_SEVERITY_FATAL
+
+    @staticmethod
+    def valid_logging_severity(level: IntEnum) -> bool:
+        if level == LoggingSeverity.UNSET \
+           or level == LoggingSeverity.DEBUG \
+           or level == LoggingSeverity.INFO \
+           or level == LoggingSeverity.WARN \
+           or level == LoggingSeverity.ERROR \
+           or level == LoggingSeverity.FATAL:
+            return True
+        else:
+            return False
```

Comment:
I think we can use catching the exception (ValueError) instead of this.

Reply:
Do you mean valid_logging_severity() throw exception instead of returning bool ?

Reply:
I mean just to remove this static method.
If users set a wrong level value into `LoggingSeverity`, `LoggingServerity` will throw a ValueError exception.

Reply:
> Enum feature of python.

Okay.